RocketChat/Rocket.Chat.ReactNative
# [FIX] Team icons on Share View and searching on RoomsListView

## Proposed changes

When opening the Share View to share some file, the icon that represents the teams are incorrect, showing the same as the channels. So we need to fix it.


## Issue(s)	


## How to test or reproduce

- Be part of a team and channel
- Go to the OS gallery
- Share something
- Notice if the icons are showing properly for each type of room

## Screenshots

#### Before
![Screen Shot 2023-01-18 at 15 13 07](https://user-images.githubusercontent.com/47038980/213261304-9c559f9c-7f35-4930-b65e-12d9310d70f7.png)



#### After
![Screen Shot 2023-01-18 at 15 11 23](https://user-images.githubusercontent.com/47038980/213260960-713937d8-9584-48a7-8de7-7a8b88c1be0b.png)


## Types of changes




## Checklist



## Further comments

[TC-328]

[TC-328]: https://rocketchat.atlassian.net/browse/TC-328?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
